src/ResourceManager/Compute/Commands.Compute/Generated/ContainerService/Config/AddAzureRmContainerServiceAgentPoolProfileCommand.cs

```diff
@@ -49,7 +49,7 @@ public class AddAzureRmContainerServiceAgentPoolProfileCommand : Microsoft.Azure
             Mandatory = false,
             Position = 2,
             ValueFromPipelineByPropertyName = true)]
-        public int? Count { get; set; }
+        public int Count { get; set; }
```

Comment:
It is changed because the base client library is changed.  The base Swagger spec is the same, but AutoRest generates a different type after we updated AutoRest version.  Now the client library has int type instead of int?